### ISSUE TITLE
vsgviewer cannot be built on MacOS

### ISSUE THREAD

**Issue by rainergericke** [vsg-dev/vsgExamples#116]
After the last commit there is a problem on MacOS:
<img width="975" alt="vsgExamples problem on MacOS 2021-10-20 um 09 22 31" src="https://user-images.githubusercontent.com/31243285/138047226-a5abc9dd-5a86-484e-a442-111895538dbf.png">

vsgviewer cannot be built due to this problem.

**Comment by robertosfield** [comment]
I checked in the required changes yesterday so you'll need to pull the latest VulkanSceneGraph master.

The commit to VSG master was: [6c979fd9795726f886afc602c939247ba0baea36](https://github.com/vsg-dev/VulkanSceneGraph/commit/6c979fd9795726f886afc602c939247ba0baea36)

**Comment by rainergericke** [comment]
You are right, but github delayed this commit, so it wasn't available at 9:30 neither via github web nor via github desktop. I was able to load it now (12:30). Doesn't matter it works now on Windows and MacOS. Thanks.
